generated/nidigital/tox-system_tests.ini

```diff
@@ -37,6 +37,7 @@ commands =
     nidigital-coverage: coverage report --rcfile=../../tools/coverage_system_tests.rc
 
 deps =
+    nidigital-wheel_dep: build
     nidigital-wheel_dep: packaging
```

Comment:
We might not need the packaging dep. I haven't tried removing it.

Reply:
try it in separate PR